tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -638,19 +638,38 @@ function configure_containerd() {
 
 	mkdir -p /etc/containerd/
 
-	if [ $use_containerd_drop_in_conf_file = "false" ] && [ -f "$containerd_conf_file" ]; then
-		# only backup in case drop-in files are not supported, and when doing the backup
-		# only do it if a backup doesn't already exist (don't override original)
-		cp -n "$containerd_conf_file" "$containerd_conf_file_backup"
-	fi
+	# To avoid problems when this pod gets re-created let's be a bit paranoiac
+	# and try to configure/backup the most recent configuration more-less
+	# atomically. This shouldn't be that needed for drop-in configuration
+	# but shouldn't harm either.
+	local tmp_config_file=$(mktemp)
+	( for i in {1..10}; do
+		local pre_config="$(cat "$containerd_conf_file")"
```

Comment:
Considering that we may do this in pod recreation, shall we use the containerd backup file previosly created instead of using this one?

Reply:
Not sure I follow. I'm always re-creating the pre_config because someone else might have modified the file while we were trying to update it, which is why we always want the latest configuration.

Reply:
Neither do I. @fidencio mind to explain again? Or maybe you are happy with these changes...